R/render.r

```diff
@@ -370,3 +373,53 @@ footer_pkgdown <- function(data) {
     data$pkgdown$version, "."
   )
 }
+
+data_deps <- function(pkg, depth) {
+
+  # theme variables from configuration
+  bs_version <- get_bs_version(pkg)
+  bootswatch_theme <- pkg$meta[["template"]]$bootswatch %||% NULL
+  bs_theme <- do.call(
+    bslib::bs_theme,
+    c(
+      list(
+        version = bs_version,
+        bootswatch = bootswatch_theme
+        ),
+      pkg$meta[["template"]]$bslib
+    )
+  )
+  deps <- bslib::bs_theme_dependencies(bs_theme)
+  # Add other dependencies - TODO: more of those?
+  # Even font awesome had a too old version in R Markdown (no ORCID)
+
+  # Dependencies files end up at the website root in a deps folder
+  deps <- lapply(
+    deps,
+    htmltools::copyDependencyToDir,
+    file.path(pkg$dst_path, "deps")
+  )
+
+  # Function needed for indicating where that deps folder is compared to here
+  transform_path <- function(x) {
```

Comment:
this and the navbar link creation might need to share logic? not sure. in any case both involve `depth`.

Reply:
Can we use an fs function for this?

Reply:
I've now read fs docs again, and I don't see any good fit.

Reply:
`fs::path_rel()` doesn't help?